brave/brave-core
# Added header to bookmarks panel

fix https://github.com/brave/brave-browser/issues/32952

Bookmarks panel's header includes icon, title and button for loading bookmarks manager.

`BraveBookmarksSidePanelCoordinator` gives bookmaks panel's contents view and that contents view
has our header view and upstream web view.

![image](https://github.com/brave/brave-core/assets/6786187/34446731-72a7-405d-91ef-3838fead720e)

![image](https://github.com/brave/brave-core/assets/6786187/a5d6e498-330c-4d24-851a-08510569fea0)



Resolves 



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Toggle bookmarks panel and check header is shown.
2. Click button in the header and check bookmarks manager is loaded in a tab
